cicd/monit_pypi/Dockerfile

```diff
@@ -0,0 +1,51 @@
+ARG BASE_TAG=latest
+FROM registry.cern.ch/cmscrab/crabtaskworker:${BASE_TAG}
+
+# RUN apt update && apt install -y \
+#     vim nano \
+#     && \
+#     apt clean && rm -rf /var/lib/apt/lists/*
+# # the following packages are necessary, but already provided inside the docker image
+# #  libcurl-openssl-dev, libssl-dev \
+
+# ENV WDIR=/data
+# ENV USER=crab3
+# # add new user and switch to user
+# RUN useradd ${USER} &&\
+#     install -o ${USER} -d ${WDIR} &&\
+#     install -o ${USER} -d /home/${USER}
+# USER ${USER}
+
+# pip is already available in the python:3.8 docker image
+# RUN python3.8 -m ensurepip
+RUN python3.8 -m pip install --user pandas 
+# not necessary: rucio-clients pycurl jwt future
+
+RUN mkdir -p /data/srv/tmp
+# WORKDIR ${WDIR}
+
+# RUN git clone https://github.com/dmwm/WMCore /data/srv/monit/WMCore
+RUN mkdir -p /data/srv/monit/
+COPY ./src/python/ServerUtilities.py           /data/srv/monit
+COPY ./src/python/RESTInteractions.py          /data/srv/monit
+COPY ./src/script/Monitor/GenerateMONIT.py     /data/srv/monit
+COPY ./scripts/Utils/CheckTapeRecall.py        /data/srv/monit
+COPY ./src/script/Monitor/aso_metrics_ora.py   /data/srv/monit
+COPY ./src/script/Monitor/ReportRecallQuota.py /data/srv/monit
```

Comment:
Please do multiline COPY like this: 

https://github.com/dmwm/CRABServer/blob/8e4ff122b8d64c4b7fb7f2594748b13b2455b91a/cicd/crabtaskworker_pypi/Dockerfile#L113-L118